app/presenters/audit_presenter.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class AuditPresenter
+  extend Forwardable
+
+  def initialize(audit, expand: false)
+    @audit = audit
+    @expand = expand
+  end
+
+  def expand?
+    @expand
+  end
+
+  def successes
```

Comment:
Here the presenter knows too much about event. Define a method `#successful_events` on `Audit`. That method should apply the `successful` scope defined on the `Event` class.   The same goes for the failures.